### DISCUSSION TITLE
Asking for TikTok live plugin

### DISCUSSION THREAD

**Discussion by aibi786** [streamlink/streamlink#5765] [General]
### Description

i tried to find TikTok live plugin for streamlink but i haven't found any
i tried to make it by myself in python but i failed cuz i don't have enough knowledge about programming

**Comment by bastimeyer** [comment]
Please read the rules before posting
https://github.com/streamlink/streamlink/blob/master/.github/DISCUSSION_TEMPLATE/general.yml

Also very much a duplicate...
https://github.com/streamlink/streamlink/issues?q=tiktok
